bin/pic-build

```diff
@@ -40,11 +40,12 @@ help()
     echo ""
     echo "usage: pic-build [OPTIONS]"
     echo ""
+    echo "-j [N]                - allow N jobs at once; infinite jobs with no arg"
```

Comment:
Maybe we should call it `unlimited jobs` instead of `infinite jobs` like in the `make` documentation? The infinite looks very weird

Reply:
Nevermind, this here is 1 to 1 what `make --help` says, I looked at the man-pages.